packages/clerk-js/src/ui/components/OrganizationProfile/InviteMembersForm.tsx

```diff
@@ -195,6 +214,7 @@ const AsyncRoleSelect = (field: ReturnType<typeof useFormControl<'role'>>) => {
       >
         <RoleSelect
           {...field.props}
+          value={field.props.value || (defaultRole ?? '')}
```

Comment:
I think think default role should simply be the initial value, when we call useFormControl

Reply:
Updated on https://github.com/clerk/javascript/pull/4210/commits/8b634a70edec0ad7f76a03e97b2ce45af42472d1

Reply:
@panteliselef I've actually reverted this previous commit initializing with `useFormControl`. 

I only noticed that the unit tests fail because of that, but the field value doesn't get updated when the roles get fetched. 

The default role gets resolved as undefined on a couple of renders until it gets resolved, but `useFormControl` doesn't seem to update its internal state. 

I've added the `value` prop back and it's re-rendering `RoleSelect` when that changes.